packages/circuit-ui/components/CardList/CardList.stories.js

```diff
@@ -35,7 +34,7 @@ export const Base = (args) => {
 
   return (
     <CardList {...args}>
-      {range(1, 6).map((i) => (
+      {Array.from(Array(5).keys()).map((i) => (
```

Comment:
Interesting, I normally spread it like:

```suggestion
      {[...Array(5).keys()].map((i) => (
```

...or we could also just do this 😄 

```suggestion
      {[0,1,2,3,4].map((i) => (
```

All approaches are fine by me

Reply:
I like the second one, it's the easiest to read.